packages/babel-types/src/definitions/core.ts

```diff
@@ -1207,19 +1207,35 @@ defineType("VariableDeclaration", {
   validate:
     process.env.BABEL_8_BREAKING || process.env.BABEL_TYPES_8_BREAKING
       ? (() => {
-          const withoutInit = assertNodeType("Identifier");
+          const withoutInit = assertNodeType("Identifier", "Placeholder");
+          const constOrLetOrVar = assertNodeType(
+            "Identifier",
+            "ArrayPattern",
+            "ObjectPattern",
+            "Placeholder",
+          );
+          const usingOrAwaitUsing = assertNodeType("Identifier", "Placeholder");
```

Comment:
```suggestion
          const usingOrAwaitUsing = withoutInit;
```

They are the same, we can avoid creating two separate functions.